### PR TITLE
Migrate article: Limit degrees of freedom in development

### LINKED ISSUE #71: Migrate article: Limit degrees of freedom in development
Part of #55. Reference PRs and Issues on how to repeat this, basically: https://github.com/ScalaWilliam/ScalaWilliam.com/issues/61 https://github.com/ScalaWilliam/ScalaWilliam.com/issues/65 https://github.com/ScalaWilliam/ScalaWilliam.com/pull/67 https://github.com/ScalaWilliam/ScalaWilliam.com/pull/62

I'll pay $15 for this. It's quite small and has two images.

Make sure to post "**I'm taking on this**" before starting to lock the piece of work to yourself. It's posted on http://work.scalawilliam.com/migrate-degrees-freedom-article/

### PR DESCRIPTION
**Pull request by Zegnat**
Closes #71.

Nothing spectacular here.

* Link within ScalaWilliam.com is now relative.
* Wrote a bare minimal `description` in the meta data.

### CODE REVIEW

**Comment by ScalaWilliam** [comment]
Merge conflict now.

**Comment by Zegnat** [comment]
Rebased on new h-feed.

Hopefully git understands that these article migrations do not conflict with each other.

File: 1612/limit-degrees-of-freedom/index.html [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+<meta property="article:published_time" content="2016-12-17">
+<meta property="article:modified_time" content="2016-12-17">
+<meta property="og:description" content="William Narmontas describes how limiting yourself to one direction can help.">
+<meta itemprop="description" content="William Narmontas describes how limiting yourself to one direction can help.">
+<meta name="description" content="William Narmontas describes how limiting yourself to one direction can help.">
+<meta name="twitter:description" content="William Narmontas describes how limiting yourself to one direction can help.">
+<meta property="og:site_name" content="Scala William">
+<link rel="author" href="https://plus.google.com/u/0/103489630517643950426/">
+<link rel="publisher" href="https://plus.google.com/u/0/103489630517643950426/">
+<meta itemprop="name" content="Why Scala didn't miss the Android opportunity">
```
Comment (ScalaWilliam) [inline 1612/limit-degrees-of-freedom/index.html]: Different article.

**Comment by Zegnat** [comment]
Fixed misnaming in one `META` element.

**Pull request merged**